verify/helpers_test.go

```diff
@@ -0,0 +1,206 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/firestore"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	ctx    context.Context
+	cancel context.CancelFunc
+	client *firestore.Client
+)
+
+func TestMain(m *testing.M) {
+	ctx, cancel = context.WithTimeout(context.Background(), 10*time.Second)
+	client = newFirestoreMockClient(ctx)
+
+	code := m.Run()
```

Comment:
was declaring and assigning the run method to a variable really necessary?

Reply:
It is not necessary, as the test will exit with a default exit code of 0 if we don't pass anything to it. But passing an exit code is recommended so that if there are any errors it would be properly reported

Reply:
sounds good.